pyproject.toml

```diff
@@ -0,0 +1,52 @@
+[build-system]
+requires = ["setuptools >= 40.0.0"]
+build-backend = "setuptools.build_meta:__legacy__"
+
+[project]
+name = "climage"
+dynamic = ["version"]
+description = "Convert images to beautiful ANSI escape codes"
+authors = [
+  { name = "Patrick Nappa", email = "patricknappa@gmail.com" },
+]
+license = { text = "MIT"}
```

Comment:
Argh, I've just noticed that this is incorrect (but not your fault, the old setup.py had this mistake too) - this has always been intended to be GPL v3, as per the LICENSE file in the repo. I'll make a follow up PR to fix this.

Reply:
Actually, nevermind. I'll keep it MIT, and fix the LICENSE file in the repo to be MIT.